src/chainparams.cpp

```diff
@@ -492,22 +509,25 @@ void CRegTestParams::UpdateActivationParametersFromArgs(const ArgsManager& args)
     for (const std::string& strDeployment : args.GetArgs("-vbparams")) {
         std::vector<std::string> vDeploymentParams;
         boost::split(vDeploymentParams, strDeployment, boost::is_any_of(":"));
-        if (vDeploymentParams.size() != 3) {
-            throw std::runtime_error("Version bits parameters malformed, expecting deployment:start:end");
+        if (vDeploymentParams.size() < 3 || vDeploymentParams.size() > 4) {
+            throw std::runtime_error("Version bits parameters malformed, expecting deployment:@start_height:@timeout_height[:@min_activation_height]");
```

Comment:
I'm not sure it's a good idea to use underscores here when everything else uses `startheight`

Reply:
Done